QUEST.tsv

```diff
@@ -2391,48 +2391,48 @@ QUEST_20150803_002398	There is a manager that manages the fortress of the great
 QUEST_20150803_002399	Leave me a note so I can also make a verification ticket.
 QUEST_20150803_002400	When the greed and the arrogance of the people reach the end, the sacred tree that sleeps underground will wake up.{nl}The sacred tree that woke up will punish the arrogant people and take away everything they've achieved.
 QUEST_20150804_002401	Nothing is written on it.
-QUEST_20150908_002402	
-QUEST_20150918_002403	
-QUEST_20150918_002404	
-QUEST_20150918_002405	
-QUEST_20150918_002406	
-QUEST_20150918_002407	
-QUEST_20150918_002408	
-QUEST_20150918_002409	Welcome. Sir Uska is waiting for the Revelator in the Central Plaza of Klaipeda.{nl}Is there something you are not quite familiar with?
-QUEST_20150918_002410	
-QUEST_20150918_002411	
-QUEST_20150918_002412	
-QUEST_20150918_002413	
-QUEST_20150918_002414	
-QUEST_20150918_002415	
-QUEST_20150918_002416	
-QUEST_20150918_002417	
-QUEST_20150918_002418	
-QUEST_20150918_002419	
-QUEST_20150918_002420	
-QUEST_20150918_002421	
-QUEST_20150918_002422	
-QUEST_20150918_002423	
-QUEST_20150918_002424	
-QUEST_20150918_002425	
-QUEST_20150918_002426	
-QUEST_20150918_002427	
-QUEST_20150918_002428 Underground Chapel Dungeon
-QUEST_20150918_002429	
-QUEST_20150918_002430	
-QUEST_20150918_002431	
-QUEST_20150918_002432	
-QUEST_20150918_002433	
-QUEST_20150918_002434	
-QUEST_20150918_002435	
-QUEST_20150918_002436	
-QUEST_20150918_002437	
-QUEST_20150918_002438	
-QUEST_20150918_002439	
-QUEST_20150918_002440	
-QUEST_20150918_002441	
-QUEST_20150918_002442	
-QUEST_20150918_002443	
-QUEST_20150918_002444	
-QUEST_20150918_002445	
-QUEST_20150918_002446	
+QUEST_20150908_002402	Obtained the woods
+QUEST_20150918_002403	There are no missions for Today.
+QUEST_20150918_002404	There are only missions that seem difficult to you.{nl} Please come again next time.
+QUEST_20150918_002405	Historic Site Ruins
+QUEST_20150918_002406	Historic Site Ruins(Recommended LV : 90). Only you and your party members can enter together.{nl}Are you sure you want to enter?
+QUEST_20150918_002407	Right. Goddess Laima is the strangest of them all.{nl}No one knows when she disappeared. To be honest , I don't know if she's really a Goddess.{nl}
```

Comment:
Change the last sentence to: "To be honest, I don't know if she really is a Goddess.{nl}"